NitroxClient/Debuggers/Drawer/Unity/RigidbodyDrawer.cs

```diff
@@ -0,0 +1,97 @@
+                NitroxGUILayout.Separator();
+                rb.mass = NitroxGUILayout.FloatField(rb.mass, VALUE_MAX_WIDTH);
+            }
+
+            using (new GUILayout.HorizontalScope())
+            {
+                GUILayout.Label("Drag", NitroxGUILayout.DrawerLabel, GUILayout.Width(LABEL_WIDTH));
+                NitroxGUILayout.Separator();
+                rb.drag = NitroxGUILayout.FloatField(rb.drag, VALUE_MAX_WIDTH);
+            }
+
+            using (new GUILayout.HorizontalScope())
+            {
+                GUILayout.Label("Angular Drag", NitroxGUILayout.DrawerLabel, GUILayout.Width(LABEL_WIDTH));
+                NitroxGUILayout.Separator();
+                rb.angularDrag = NitroxGUILayout.FloatField(rb.angularDrag, VALUE_MAX_WIDTH);
+            }
+
+            using (new GUILayout.HorizontalScope())
+            {
+                GUILayout.Label("Use Gravity");
+                NitroxGUILayout.Separator();
+                rb.useGravity = NitroxGUILayout.BoolField(rb.useGravity);
+            }
+
+            using (new GUILayout.HorizontalScope())
+            {
+                GUILayout.Label("Is Kinematic");
+                NitroxGUILayout.Separator();
+                rb.isKinematic = NitroxGUILayout.BoolField(rb.isKinematic);
+            }
+
+            GUILayout.Space(10);
+
+            using (new GUILayout.HorizontalScope())
+            {
+                GUILayout.Label("Interpolate", GUILayout.Width(LABEL_WIDTH));
+                NitroxGUILayout.Separator();
+                rb.interpolation = NitroxGUILayout.EnumPopup(rb.interpolation, VALUE_MAX_WIDTH);
+            }
+
+            GUILayout.Space(10);
+
+            using (new GUILayout.HorizontalScope())
+            {
+                GUILayout.Label("Collision Detection", GUILayout.Width(LABEL_WIDTH));
+                NitroxGUILayout.Separator();
+                rb.collisionDetectionMode = NitroxGUILayout.EnumPopup(rb.collisionDetectionMode, VALUE_MAX_WIDTH);
+            }
+
+            GUILayout.Space(10);
+
+            using (new GUILayout.HorizontalScope())
+            {
+                GUILayout.Label("Freeze Rotation");
+                NitroxGUILayout.Separator();
+                rb.freezeRotation = NitroxGUILayout.BoolField(rb.freezeRotation);
+            }
+
+            using (new GUILayout.HorizontalScope())
+            {
+                GUILayout.Label("Velocity", NitroxGUILayout.DrawerLabel, GUILayout.Width(LABEL_WIDTH));
+                NitroxGUILayout.Separator();
+                rb.velocity = VectorDrawer.DrawVector3(rb.velocity, VALUE_MAX_WIDTH);
+            }
+
+            using (new GUILayout.HorizontalScope())
+            {
+                GUILayout.Label("Angular Velocity", NitroxGUILayout.DrawerLabel, GUILayout.Width(LABEL_WIDTH));
+                NitroxGUILayout.Separator();
+                rb.angularVelocity = VectorDrawer.DrawVector3(rb.angularVelocity, VALUE_MAX_WIDTH);
+            }
```

Comment:
Made these be read-only.

Reply:
Making `DrawVector3Label()` wasn't necessary :D Since vector is a struct you just need to change

`rb.velocity = VectorDrawer.DrawVector3(rb.velocity, VALUE_MAX_WIDTH);`
to
`VectorDrawer.DrawVector3(rb.velocity, VALUE_MAX_WIDTH);`

Reply:
Are you sure we want to have editable textboxes that can't be edited?

Reply:
Yes. It's at least better then the current label and used in NitroxTransformDrawer.

Reply:
Done